code/datums/diseases/advance/symptoms/stimulant.dm

```diff
@@ -1,38 +0,0 @@
-/*
-//////////////////////////////////////
-
-Stimulant //gotta go fast
-
-	Noticable.
-	Lowers resistance significantly.
-	Decreases stage speed moderately..
-	Decreases transmittablity tremendously.
-	Moderate Level.
-
-Bonus
-	The body generates Ephedrine.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/stimulant
-
-	name = "Stimulant"
-	stealth = -1
-	resistance = -3
-	stage_speed = -2
-	transmittable = -4
-	level = 3

```

Comment:
> If people think stim is so OP why not at least try making it a level 8 symptom before removing it entirely

Because it's the way it affects the game. By potentially spreading a massive movespeed increase and stun resistance amongst the crew we get into situations where fundamental parts of gameplay for SS13 are affected, IE: The stun based combat. The other aspect is the fact that it is indefinite in duration. A pill of ephedrine is not particularly damning on the system, as there is an extreme potential for it to be entirely useless when used in context due to timing, etc. A stimulant virus is on 24/7 and drags down the game as a whole.

Not that SS13's ridiculous stun base combat is good, but having something in the game that permanently fucks with the game state in that fashion (on top of just being a fucking boring mechanic) isn't kosher.